data-prepper-api/src/main/java/com/amazon/dataprepper/metrics/PluginMetrics.java

```diff
@@ -2,6 +2,8 @@
 
 import com.amazon.dataprepper.model.configuration.PluginSetting;
 import java.util.StringJoiner;
+import java.util.function.ToDoubleFunction;
+
 import io.micrometer.core.instrument.Counter;
 import io.micrometer.core.instrument.DistributionSummary;
 import io.micrometer.core.instrument.Metrics;
```

Comment:
nit: Add a java doc on this class and link it micrometer docs. (Apologies for mentioning this in this PR, missed it the original PR that introduced this class)

Reply:
Sure.